src/Okta.Sdk.IntegrationTests/AuthorizationServersScenarios.cs

```diff
@@ -0,0 +1,910 @@
+        public async Task CreateAuthorizationServerPolicy()
+        {
+            var testClient = TestClient.Create();
+            var testAuthorizationServerName = $"{SdkPrefix}:Test AuthZ Server ({TestClient.RandomString(4)})";
+
+            var testAuthorizationServer = new AuthorizationServer
+            {
+                Name = testAuthorizationServerName,
+                Description = "Test Authorization Server",
+                Audiences = new string[] { "api://default" },
+            };
+            var testPolicy = new Policy
+            {
+                Type = PolicyType.OauthAuthorizationPolicy,
+                Status = "ACTIVE",
+                Name = "Test Policy",
+                Description = "Test policy",
+                Priority = 1,
+                Conditions = new PolicyRuleConditions
+                {
+                    Clients = new ClientPolicyCondition
+                    {
+                        Include = new List<string> { "ALL_CLIENTS" },
+                    },
+                },
+            };
+
+            var createdAuthorizationServer = await testClient.AuthorizationServers.CreateAuthorizationServerAsync(testAuthorizationServer);
+            var createdPolicy = await createdAuthorizationServer.CreatePolicyAsync(testPolicy);
+            try
+            {
+                createdAuthorizationServer.Should().NotBeNull();
+                createdPolicy.Should().NotBeNull();
+                createdPolicy.Name.Should().Be(testPolicy.Name);
+                createdPolicy.Description.Should().Be(testPolicy.Description);
+            }
+            finally
+            {
+                await createdAuthorizationServer.DeletePolicyAsync(createdPolicy.Id);
+                await createdAuthorizationServer.DeactivateAsync();
+                await testClient.AuthorizationServers.DeleteAuthorizationServerAsync(createdAuthorizationServer.Id);
+            }
+        }
+
+        [Fact(Skip = "OKTA-303121: Api validation failed: com.saasure.framework.exception.KeyStoreLimitExceededException (400, E0000001): Unable to create new authorization server. You cannot create authorization servers because signing keys could not be generated. To fix this issue, contact support.")]
+        public async Task UpdateAuthorizationServerPolicy()
+        {
+            var testClient = TestClient.Create();
+            var testAuthorizationServerName = $"{SdkPrefix}:Test AuthZ Server ({TestClient.RandomString(4)})";
+
+            var testAuthorizationServer = new AuthorizationServer
+            {
+                Name = testAuthorizationServerName,
+                Description = "Test Authorization Server",
+                Audiences = new string[] { "api://default" },
+            };
+            var testPolicy = new OAuthAuthorizationPolicy
+            {
+                Name = $"{SdkPrefix}:Test Policy",
+                Type = PolicyType.OauthAuthorizationPolicy,
```

Comment:
We should rename `OauthAuthorizationPolicy` to `OAuthAuthorizationPolicy` to be consistent with the policy name.

Reply:
That is a pre-existing type from a couple years ago.  Wasn't sure if changing it would cause issues if it is already referenced outside of what we can see.

Reply:
We have renamed other stuff as well keeping in mind that all SDKs will release a major version.